ue4docker/infrastructure/WindowsUtils.py

```diff
@@ -142,22 +112,18 @@ def getDllSrcImage(basetag: str) -> str:
 		return f'mcr.microsoft.com/windows:{tag}'
 
 	@staticmethod
-	def getValidBaseTags() -> [str]:
-		'''
-		Returns the list of valid tags for the Windows Server Core base image, in ascending chronological release order
-		'''
-		return WindowsUtils._validTags
-
-	@staticmethod
-	def isValidBaseTag(tag: str) -> bool:
+	def getKnownBaseTags() -> [str]:
 		'''
-		Determines if the specified tag is a valid Windows Server Core base image tag
+		Returns the list of known tags for the Windows Server Core base image, in ascending chronological release order
 		'''
-		return tag in WindowsUtils._validTags
+		return WindowsUtils._knownTags
 
 	@staticmethod
-	def isNewerBaseTag(older: str, newer: str) -> bool:
+	def isNewerBaseTag(older: str, newer: str) -> Optional[bool]:
```

Comment:
Non-blocking nit-pick: I'm reasonably unfond of `Optional[bool]`, but don't have a better option, except letting the `ValueError` bubble up, or throwing a custom exception, and catching that in the caller.

Reply:
Dunno, these both look equally ugly to me :D

I'd actually make a enum { Newer, Older, Unknown } but don't think it will improve anything at all given that we're in Python that ~doesn't even have switch/case~. Okay, it got it in 3.10, but that's too much new yet to use.